python/example_code/glue/flight_etl_job_script.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script is used by the AWS Glue _getting started with crawlers and jobs_ scenario to
+perform extract, transform, load (ETL) operations on sample flight data.
```

Comment:
extract, transform, load (ETL) --> extract, transform, and load (ETL)

Reply:
fixed